ts/Dashboards/Actions/DragDrop.ts

```diff
@@ -595,7 +595,7 @@ class DragDrop {
         mouseRowContext: Row
     ): void {
         const dragDrop = this,
-            dropPointerSize = dragDrop.options.dropPointerSize,
+            dropPointerSize = dragDrop.options.dropPointerSize || 0,
```

Comment:
What is the reason of `0` placed in many params? The `dragDrop.options.dropPointerSize` can be declared in defaultOptions or replaced with no-optional variable.

Reply:
All the parameters here, among them `dropPointerSize`, should be optional. The default value is declared, the `|| 0`is only to trick the TS here 😉